README.md

```diff
@@ -46,10 +46,18 @@ Retrieve by name if you know the display name:
 $ az ad group list --query "[?displayName=='dcd_devops'].{DisplayName: displayName, ObjectID: objectId}" -o table
 ```
 
-### managed_identity_object_id
-If your application is running in kubernetes it will retrieve the secrets with a managed identity
+### managed_identity_object_ids
+If your application is running in kubernetes it will retrieve the secrets with a managed identity.
 
-In order to allow the managed identity access you need to add an additional variable to the module (`managed_identity_object_id`)
+In order to allow the managed identity access you need to add an additional variable to the module (`managed_identity_object_ids`).
+```
```

Comment:
best to add an example of creating the identity as well I think, not sure if we want that to be a module or not, it only creates one resource